### DISCUSSION TITLE
Started to work on ML4W Dotfiles AUR

### DISCUSSION THREAD

**Discussion by mylinuxforwork** [mylinuxforwork/dotfiles#153] [Announcements]
Dear ML4W community. I hope this is good news. I have started to work on PKGBUILD files to offer the ML4W Dotfiles in the future as AUR package.

The plan is to provide 2 AUR package:
ml4w-dotfiles-git with the latest commits (rolling release)
ml4w-dotfiles with the latest tagged version

As AUR, the ML4W Dotfiles and dependecies (except AUR dependencies) can be installed on your system with any AUR helper like yay or paru. To finalize then the installation, you just need ml4w-dotfiles into your terminal.

Of course, it will again require some adjustments on the directory structure.

What do you think about it?

**Comment by kernelpanic95** [comment]
I think it'll be lovely to have an AUR package. Very good idea!

**Comment by hambosto** [comment]
Can't Wait🙌

**Comment by Homer-Max** [comment]
support you

**Comment by bknightInfo** [comment]
Can't wait to test this

**Comment by ivanMosquedaFTP** [comment]
why was it removed from the AUR?

**Reply by Aiakio** [reply]
Because the main way to install these dotfiles now is the flatpak installer.
I assume one flatpak is easier to maintain than a dedicated package for each distro. Since ml4w dotfiles are available for a wide range of distros, this makes sense to me.

**Comment by SSYH** [comment]
I have successfully invoked "paru ml4w-hyprland" for many months.
(with the occasional "ml4w-hyprland-setup" when reminded.)
This recently disappeared. 
Is that because of this shift?
What is the preferred install invocation now?
Thx
